src/command.js

```diff
@@ -11,6 +11,12 @@ import pkg from '../package.json';
 
 // used to deserialize regular expression strings
 const RE_REGEXP = /^\/(.+)\/(\w+)?$/;
+const WAIT_FOR_TIMEOUT = 5000;
```

Comment:
Last small pedantic thing, I promise. 

I feel like this name is a little vague (wait for what?). How about `EVAL_TIMEOUT`?